src/dialogs/config-entry-system-options/dialog-config-entry-system-options.ts

```diff
@@ -72,7 +72,7 @@ class DialogConfigEntrySystemOptions extends LitElement {
                     `
                   : ""}
                 <div class="form">
-                  <paper-toggle-button
+                  <ha-switch
                     .checked=${!this._disableNewEntities}
                     @checked-changed=${this._disableNewEntitiesChanged}
```

Comment:
This event is not fired by `mwc-switch`. Change to `@change`. Also `this._disableNewEntitiesChanged` should be changed as it would not work now.

Reply:
Styling is also a bit weird now:
![image](https://user-images.githubusercontent.com/5662298/65353550-b7dda880-dbed-11e9-8f30-78593a9a6423.png)